### PR TITLE
[DMS] Merging pg mysql from extension into core

### PR DESCRIPTION
**Pull request by temandr**
**Description**  

* Merging MySQL and PostgreSQL migration capability from the extension into the core. Since these scenarios have been in GA for a while, this change allows the customer to perform MySQL and PostgreSQL migrations (as well as cutover the migration) from the core cli package. 

**Testing Guide**  

No new tests added. Modified existing tests to test the scenario inputs for the MySQL and PostgreSQL scenarios. 

**History Notes**  

[DMS] BREAKING CHANGE: az dms project task create: Added mandatory task type parameter to help distinguish if a scenario is an online migration or an offline migration.  
[DMS] az dms project task cutover: add new command which allows tasks with an online migration task type to cutover and end the migration.
[DMS] az dms project create/az dms project task create: Enable MySQL and PostgreSQL projects/tasks to be created.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
DataMigration

**Comment by temandr** [comment]
@yonzhan This is now ready for review. Please let me know if there are any questions or concerns.

**Comment by yonzhan** [comment]
@evelyn-ys will help you with this PR.

File: src/azure-cli/azure/cli/command_modules/dms/custom.py
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/dms/custom.py:165]: Can `project_show` return `task_type` along with source_platform and target_platform? In that case, we can use project's task_type as the default value of the param `task_type` for task creating. And there will be no breaking change here since take_type is no longer mandatory

File: src/azure-cli/azure/cli/command_modules/dms/custom.py
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/dms/custom.py:169]: Can `project.get` return `task_type` along with source_platform and target_platform?
- If it can return, can we use project's task_type as the default value for task creating? 
- If it can't return, can we default task_type to `""`? If passed `""`, `core_handles_scenario` can fill with default task_type for each scenario.
And `task_type` can be no longer mandatory in such ways so there will be no breaking change.

Comment (temandr) [reply src/azure-cli/azure/cli/command_modules/dms/custom.py]: Project task_type is currently unreliable so set to default.

**Review by evelyn-ys — APPROVED** [review]

**Comment by evelyn-ys** [comment]
This pr may need more approvals. @Juliehzl Could you help review?

**Comment by Juliehzl** [comment]
I am not the code owner. My approval is not useful. Please add codeowner in .github and then this PR could be merged.

File: src/azure-cli/azure/cli/command_modules/dms/_help.py
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/dms/_help.py:64]: prefer previous help message. `Task` should be used at the beginning of the sentence.
The same for above changes

File: src/azure-cli/azure/cli/command_modules/dms/custom.py
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/dms/custom.py:193]: why removing these code?

**Comment by Juliehzl** [comment]
Hi @temandr, may I know which extension is used for these commands previously? We may need to suppress the extension usage after migration.

File: src/azure-cli/azure/cli/command_modules/dms/custom.py
Comment (temandr) [reply src/azure-cli/azure/cli/command_modules/dms/custom.py:193]: This was refactored into custom.create_connection.

**Comment by temandr** [comment]
> ?

https://github.com/Azure/azure-cli-extensions/tree/master/src/dms-preview
Although not all features are being migrated; so, after this is checked in, I will work on removing the migrated features.

**Comment by Juliehzl** [comment]
@msyyc as you are the code owner, we need your approval to merge the PR.

**Review by msyyc — APPROVED** [review]

**Pull request merged**
